sycl/include/CL/sycl/accessor.hpp

```diff
@@ -292,6 +293,12 @@ class image_accessor
   static_assert(Dimensions > 0 && Dimensions <= 3,
                 "Dimensions can be 1/2/3 for image accessor.");
 
+  template <info::device param>
+  void check_device_feature_supported(const device &Device) {
+    if (!Device.get_info<param>())
+      throw feature_not_supported("Images don't support by this device");
```

Comment:
"Images are not supported by this device" or "Images aren't supported by this device" may be better English in this case

Reply:
Done.